EliteDangerous/EliteDangerous/Translations.cs

```diff
@@ -603,12 +604,12 @@ internal enum EDCTx
         JournalCarrier_ReserveBalance, 
         JournalCarrier_AvailableBalance,
         JournalCarrier_ReservePercent,
-        JournalCarrier_TaxRatePioneersupplies,
-        JournalCarrier_TaxRateShipyard,
-        JournalCarrier_TaxRateRearm,
-        JournalCarrier_TaxRateOutfitting,
-        JournalCarrier_TaxRateRefuel,
-        JournalCarrier_TaxRateRepair,
+        JournalCarrier_TaxRatePioneersupplies, // Tax Rate Pioneersupplies: ;;N1
```

Comment:
why has this been delta?

Reply:
Forgot to write down the commented thing. As most of the things have it I added it.